frontend/src/v5/store/tickets/card/ticketsCard.selectors.ts

```diff
@@ -33,12 +33,18 @@ export const selectCurrentTickets = createSelector(
 	selectTickets,
 );
 
-const ticketToPin = (ticket:ITicket, selectedId): IPin => ({
+export const selectCurrentTemplates = createSelector(
+	(state) => state,
+	selectCurrentModel,
+	selectTemplates,
+);
```

Comment:
You can't use selectedTemplate here because selectTicketPins is in use when in the tickets list. It shows the default pins of all the tickets which may depend on different templates